app.py

```diff
@@ -177,7 +177,7 @@ def check_auth():
     connection_class=elasticsearch.RequestsHttpConnection
 )
 app = Chalice(app_name='dos-azul-lambda')
-app.debug = os.environ.get('DEBUG', False) == 'True'
+app.debug = os.environ.get('DEBUG', True) == 'True'
```

Comment:
Note that if the `DEBUG` environment variable does not exist, this statement will still evaluate to False. This is because the boolean `True` is not equal to the string `True`.